olclient/cli.py

```diff
@@ -61,18 +65,19 @@ def main():
     args = parser.parse_args()
 
     if args.configure:
-        username = args.username
-        if not username:
-            raise ValueError("--username required for configuration")
+        email = args.email
```

Comment:
We might want to save the email in the config (it's useful for looking at the config values and seeing what account the s3 keys belong to)